woocommerce/woocommerce-ios
# REST API: Migrate system status endpoints

Closes: #8602 


## Description

This PR continues the migration for the REST API project, this time is for system status endpoints:
- Updated the mappers for system status and system plugins.
- Enable REST API for system status endpoints.

## Testing instructions


- Enable the feature flag `applicationPasswordAuthenticationForSiteCredentialLogin` and build the app.
- Log out of the app or skip onboarding if needed.
- On the prologue screen, select Enter your site address and proceed with the address of your self-hosted site.
- Proceed to log in with your site credentials.
- After the login succeeds, you should be navigated to the home screen.
- Navigate to the Menu app and select Settings.
- Notice on the row "WooCommerce Version", the correct version for Woo on your site should be displayed.
- Select "Help & Support" > "System Status Report". The report should be loaded and displayed without error.


## Screenshots

N/A

---
